src/components/clusterDetail/ClusterNetworkDetails.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Flex,
+} from '@patternfly/react-core';
+import { getCamelCasedClusterObject } from './utils';
+import { Cluster } from '../../api/types';
+
+type ClusterMetworkDetailsProps = {
+  cluster: Cluster;
+};
+
+const ClusterNetworkDetails: React.FC<ClusterMetworkDetailsProps> = ({ cluster }) => {
+  const clusterObj = getCamelCasedClusterObject(cluster);
```

Comment:
I think we should not mix our components with such case conversions. The component expects property of the `Cluster` type, which is already camel-cased.
Our axios client has already middleware performing such conversion.

In case the component is instantiated from an external context which is unaware of typescript, we should either convert it there or pass just JSON here, let's decide after considering performance implications.

Another solution can be in wrapping by an HOC component providing such conversion.

Reply:
I forgot to remove this file. we don't need it.